test/tools/integration/provision_master.sh

```diff
@@ -59,6 +59,7 @@ if [[ "${1:-deploy_machine_controller}"  == "do-not-deploy-machine-controller" ]
 fi
 if ! ls machine-controller-deployed; then
   docker build -t kubermatic/machine-controller:latest .
+  sed -i -e 's/-worker-count=5/-worker-count=20/g' machine-controller.yaml
```

Comment:
This is a noop because the `examples/machine-controller.yaml` file does not contain a `-worker-count=5` string but instead `-worker-count=20`  - Is it intended to specify 20 workers in the example file?

Reply:
Not intended. For the tests though very helpful